src/OEBPS/Chapter21.xhtml

```diff
@@ -268,7 +268,7 @@
             <p>Harry hauled himself back up onto his broom, just in time for a Bludger to glance off his hip and send him spinning. The Slytherin beaters were closing in with blood in their eyes. This made sense, because if the Snitch appeared while Draco was taking a dirt-nap, they had no prayer of winning.</p>
             <p>The sensibility of their actions did nothing to deter Harry’s anger though. As soon as he got his broom back under control he shot up toward the Slytherin beaters, going almost as fast as the broom could go. The one on the right, Peregrine Derrick, was closing in on an arcing Bludger, bat poised to strike. Harry gritted his teeth and shot almost directly toward the thick-set boy. As the beater closed in on the Bludger and cocked his arm back to hammer it with his enchanted bat, Harry veered off at high speed.</p>
             <p>Derrick got a nasty grin as he struck the iron ball with a resounding smack. He waited until Harry was right next to him to maximize the damage. When Harry saw the boy’s arm begin to move, he opened the broom up the rest of the way, gaining a tiny increase in speed.</p>
-            <p>This was enough to make the Bludger brush the back of Harry’s robes instead of staving in his ribcage. There was a nasty crunching sound a moment later though. Derrick’s partner, Bole, was directly in the path of the hard-struck Bludger, and it nailed the boy dead-centre in the chest. Bole flew backwards off his broom and plummeted toward the ground before someone caught him with a levitation charm.</p>
```

Comment:
No, "staving" is legitimate. See https://www.dictionary.com/browse/staving

In this context, it basically means punching a hole in his ribcage and letting his insides out like a barrel of wine being opened.